_sub/network/aws-lb-controller/versions.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_version = ">= 1.6.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 6.16.0"
+    }
+    github = {
+      source  = "integrations/github"
+      version = "~> 6.6.0"
+    }
+    kubectl = {
```

Comment:
Do you even use this provider here? Did you just copy the module from somewhere else?